HEC.FDA.Statistics/Histograms/DynamicHistogram.cs

```diff
@@ -163,23 +163,7 @@ public void ForceDeQueue()
             //do nothing
             //HACK
         }
-        public double HistogramMean()
```

Comment:
Need to revisit this. This was not being used to get HistogramMeans, instead the property "Mean" on this object is calculated as observations are added to the histograms, and used the actual observation value, not the bin value. Meaning Property mean represents the population mean, not the histogram mean.

Reply:
Histogram class now has both, and clarifies better.